README.md

```diff
@@ -94,15 +94,37 @@ export class MyComponent implements OnInit {
 1. each endpoint must have an `operationId` defined
 1. `in: header` definitions are ignored
 1. `get` and `delete` methods do not contain `body`
+1. swagger file should contain values for the keys `host` and `basePath` so that each generated service method 
+can contain a link to the swagger UI method reference, e.g. `http://example.com/swagger/swagger-ui.html#!/Order/Order`
 
 ## Development
 
+* at least node 8 is needed
+
 ### Docker image
 
 1. `docker build . -t swagger-angular-generator`
 1. `docker run -u $(id -u) -it -v "$PWD":/code swagger-angular-generator bash`
 1. `npm i`
 
+### Testing
+
+##### How the testing works:
+
+* tests are written in the demo-app
+* the test swagger files can be found in demo-app/client/test-swaggers
+* upon these swagger files, interfaces and services are generated
+* the generated services are manually imported to the app.module.ts
+* unit tests can be found in demo-app/client/src/tests
+
+##### Running the tests:
+
+1. `cd demo-app/client`
+1. `npm run generate-from-swagger`
+1. `npm run test`
+
+or instead of step 2 and 3 run: `npm run testci`
+
 ---
 
-### _Pull requests are welcome!_
+### _Pull requests are welcomed!_
```

Comment:
No need for a change, *sth is welcome (adj)* if both correct and more frequent.

Reply:
Me: _would you say Pull requests are welcomed, or just Pull requests are welcome? Both correct, or just one of them?_
Native brit: _both sound ok to me. (Maybe I'd choose the second, but a quick search of Google as corpus has lots of results for both.)_